cpp/arcticdb/version/version_map_batch_methods.hpp

```diff
@@ -51,7 +51,7 @@ inline std::vector<folly::Future<version_store::UpdateInfo>> batch_get_latest_un
                                                      version_map,
                                                      stream_id,
                                                      LoadParameter{LoadType::LOAD_LATEST_UNDELETED}})
-        .thenValue([](auto&& entry){
+        .thenValue([&stream_id](auto&& entry){
```

Comment:
This seems to be unused

Reply:
yes, right. It was probably necessary in another commit and I forgot to remove it when it was not longer used.